docs/examples/contribute/contribute.jl

```diff
@@ -0,0 +1,44 @@
+# ## Contribute to Documentation
+# Contributing with examples can be done by first creating a new file example
+# [here](https://github.com/JuliaDataCubes/YAXArrays.jl/tree/master/docs/examples/UserGuide)
+
+# !!! info "new file"
+#     - `your_new_file.jl` at `docs/examples/UserGuide/`
+
+# Once this is done you need to add a new entry [here](https://github.com/JuliaDataCubes/YAXArrays.jl/blob/master/docs/mkdocs.yml)
+# at the bottom and the appropriate level.
+
+# !!! info "add entry to docs"
+#     Your new entry should look like:
+#     - `"Your title example" : "examples/generated/UserGuide/your_new_file.md"`
+
+# ## Build docs locally
+# If you want to take a look at the docs locally before doing a PR
+# follow the next steps:
+
+# !!! warning "build docs locally"
+#     Install the following dependencies in your system via pip, i.e.
+#     - `pip install mkdocs pygments python-markdown-math`
+#     - `pip install mkdocs-material pymdown-extensions mkdocstrings`
+#     - `pip mknotebooks pytkdocs_tweaks mkdocs_include_exclude_files jinja2 mkdocs-video`
+
+# Then simply go to your `docs` env and activate it, i.e.
+
+# `docs> julia`
+
+# `julia> ]`
+
+# `(docs) pkg> activate .`
+
+# Next, run the scripts:
+# !!! info
+#     Generate files and build docs by running:
```

Comment:
Does this mean include("genfiles.jl") or should I do 'julia genfiles.jl'?
Maybe you could clarify this here.

Reply:
from the repl is include("genfiles.jl") but usually just select all lines and do the run interactive.